src/signs/ui/search/MesZLContent.tsx

```diff
@@ -40,51 +46,57 @@ export default function MesZlContent({
   const [mesZlHead, setMesZlHead] = useState<string>('')
   const [mesZlBody, setMesZlBody] = useState<string>('')
 
+  const [ready, setReady] = useState(false)
+
+  const { mesZlHeadMarkdown, mesZlBodyMarkdown } = splitMesZl(mesZl)
+
   useEffect(() => {
     ;(async () => {
-      const mesZlSplitted = splitMesZl(mesZl)
-      const mesZlHeadConverted = await convertMarkdownToHtml(
-        mesZlSplitted.mesZlHead
-      )
-      const mesZlBodyConverted = await convertMarkdownToHtml(
-        mesZlSplitted.mesZlBody
-      )
-      setMesZlBody(mesZlBodyConverted)
-      setMesZlHead(mesZlHeadConverted.replace(/\s/g, '&#9'))
+      setMesZlHead(await convertMarkdownToHtml(mesZlHeadMarkdown))
+      setMesZlBody(await convertMarkdownToHtml(mesZlBodyMarkdown))
+
+      setReady(true)
     })()
-  }, [mesZl])
+  }, [mesZl, mesZlHeadMarkdown, mesZlBodyMarkdown])
 
-  if (mesZlHead && mesZlBody) {
+  if (ready) {
     return (
       <>
         <div>
           <pre>
             <div
+              style={{ fontFamily: 'Assurbanipal' }}
               className="text-center"
```

Comment:
@jlaasonen  i will change it to a css class but i don't know why it isn't inheriting it from the body currently.

Reply:
Maybe `pre` has overriding styles? You should be able to see where the final styles come from with the browser's dev tools.